recipes/genometools-genometools/meta.yaml

```diff
@@ -1,28 +1,36 @@
+{% set name = "genometools" %}
+{% set version = "1.5.10" %}
+{% set sha256 = "a6aa7f158a3cef90fea8d0fe24bfad0c3ee96b17b3ba0c1f6462582593af679e" %}
+
+
 package:
-  name: genometools-genometools
-  version: 1.5.9
+  name: {{ name }}-{{ name }}
+  version: {{ version }}
 
 source:
-  fn: genometools-1.5.9.tar.gz
-  url: https://github.com/genometools/genometools/archive/v1.5.9.tar.gz
-  sha1: 733530dfe875cd058da0c3e1b6530b9e085517f4
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
 
 requirements:
   build:
-    - gcc
+    - gcc  # [not osx]
+    - llvm  # [osx]
     - python
-    - pango
+    - pango  # [not osx]
```

Comment:
out of interest why does cairo not work for OSX?

Reply:
Cairo works, but we'd also need xquartz, which isn't currently in conda-forge:
https://github.com/genometools/genometools/blob/master/INSTALL#L51

In general, is this the right way to provide build variants (here one not requiring cairo/pango/X) of recipes, as subdirs of the main recipe, with the same version but different build string?  Conda also has 'features' but hardly any recipe seems to use that.

Reply:
features are really broken and should not be used. I'm fine with merging it, was just curious.